doc/classes/MultiMesh.xml

```diff
@@ -90,10 +90,10 @@
 	<members>
 		<member name="buffer" type="PackedFloat32Array" setter="set_buffer" getter="get_buffer" default="PackedFloat32Array()">
 		</member>
-		<member name="color_array" type="PackedColorArray" setter="_set_color_array" getter="_get_color_array" is_deprecated="true">
+		<member name="color_array" type="PackedColorArray" setter="_set_color_array" getter="_get_color_array" deprecated="">
 			See [method set_instance_color].
```

Comment:
Could also be the deprecation message.

This brings the question of how should deprecated properties and methods be documented in general though, I'll bring this up in the main PR comments.